static/canjs.js

```diff
@@ -87,6 +87,7 @@ function init() {
 	buildTOC();
 	setNavToggleListener();
 	setScrollPosition();
+	scrollToCurrentMenuItem();
```

Comment:
I think this should only be run when the page first loads, not in `init`. As-is, the navigation will change as people click on different links, but I think we only want this behavior on the first page load.